### ISSUE TITLE
Exception while handling OAuth2 callback (Unable to initialize due to invalid secret key). Redirecting to facebook connection status page.

### ISSUE THREAD

**Issue by brunjick** [kevindeyne/simple-facebook-integration-spring-sec#3]
After updating dependencies (#2) this exception showed up
```
Exception while handling OAuth2 callback (Unable to initialize due to invalid secret key). Redirecting to facebook connection status page.
```
It works fine if I use `Encryptors.noOpText()` while initializing `JdbcUsersConnectionRepository` [here](https://github.com/kevindeyne/simple-facebook-integration-spring-sec/blob/master/src/main/java/com/kevindeyne/web/config/WebSecurityConfig.java#L96)

What does this encryptor do? (as you see I'm newbie to Spring)

**Comment by kevindeyne** [comment]
Facebook seems to be complaining about the secret key. Can you ensure you've replaced the key in the application.properties with your own? 

Eg: 
spring.social.facebook.appId=YOUR_APP_ID
spring.social.facebook.appSecret=YOUR_APP_SECRET

The encryptors provide support for encrypting the authorization credentials provided through that connection service so they are not stored in plain-text. This should occur automatically by spring itself (but it has been more than a year since I've ran this project, so I'm not entirely sure if the latest spring has changed this in any way).

**Comment by brunjick** [comment]
Yes I have replaced credentials and it does work with no encryption. It's also strange to me what does database encryption has to do with handling response from facebook. Maybe exception is misleading. I can submit a pr if I manage fix it.
